lib/_http_incoming.js

```diff
@@ -92,6 +92,7 @@ Object.defineProperty(IncomingMessage.prototype, 'connection', {
   }
 });
 
+
```

Comment:
I think this space is spurious.

Reply:
matched double spacing in other parts of file while i was rebasing out the helper, removing